File: SoloLearn/Balconies/Balconies.java
import java.util.Scanner;

public class Balconies {

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        String[] a = input.nextLine().split(","), b = input.nextLine().split(",");
        int ha = Integer.parseInt(a[0]), la = Integer.parseInt(a[1]), hb = Integer.parseInt(b[0]), lb = Integer.parseInt(b[1]);
        if (ha * la < hb * lb)
            System.out.println("Apartment B");
        else 
            System.out.println("Apartment A");
        input.close();
    }
} 

File: SoloLearn/Balconies/Balconies.cpp
#include <iostream>
using namespace std;

int main() {
    int h1, l1, h2, l2;
    scanf("%d,%d", &h1, &l1);
    scanf("%d,%d", &h2, &l2);
    if (h1 * l1 < h2 * l2)
        cout << "Apartment B";
    else 
        cout << "Apartment A";
    return 0;
} 